frontend/test/metabase/scenarios/admin/databases/list.cy.spec.js

```diff
@@ -18,7 +18,7 @@ describe("scenarios > admin > databases > list", () => {
 
     cy.visit("/admin/databases");
 
-    cy.wait("@failedGet");
+    cy.wait(["@failedGet", "@failedGet"]);
```

Comment:
Fix the test always pass even before coding. I'm not entirely sure why there're 2 calls to `api/database`

<img width="766" alt="image" src="https://user-images.githubusercontent.com/1937582/162916595-96f35f37-feeb-4d1b-aa2c-f4567885001a.png">